src/pro/sky/java/course1/HW12/Book.java

```diff
@@ -0,0 +1,35 @@
+package pro.sky.java.course1.HW12;
+
+public class Book {
+    private String name;
+    private String author;
+    private int year;
+
+    public Book(String name, String author, int year) {
+        if (author.isEmpty()) {
+            this.name = name;
+            this.author = "Автор не указан";
+            this.year = year;
+        } else {
+            this.name = name;
+            this.author = author;
+            this.year = year;
+        }
```

Comment:
```suggestion
        if (author.isEmpty()) {
            this.author = "Автор не указан";
        } else {
            this.author = author;
        }
        this.name = name;
        this.year = year;
```
вот так будет без дублирования кода